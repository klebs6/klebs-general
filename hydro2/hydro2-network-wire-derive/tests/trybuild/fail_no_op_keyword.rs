// ---------------- [ File: tests/trybuild/fail_no_op_keyword.rs ]
// tests/trybuild/fail_no_op_keyword.rs

use hydro2_network_wire_derive::NetworkWire;

#[derive(NetworkWire)]
#[available_operators(foo="Bar")]
pub struct MyWire;

fn main() {}
