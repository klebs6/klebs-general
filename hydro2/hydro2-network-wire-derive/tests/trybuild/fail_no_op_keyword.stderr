error: Expected `op`, found `foo`
 --> tests/trybuild/fail_no_op_keyword.rs:7:23
  |
7 | #[available_operators(foo="Bar")]
  |                       ^^^
